device/Microsoft.Azure.Devices.Client/ProductInfo.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.Azure.Devices.Client
+{
+    class ProductInfo
+    {
+        public string Extra { get; set; } = "";
+
+        public override string ToString()
+        {
+            const string name = "Microsoft.Azure.Devices.Client";
+#if PCL
+            // DO NOT EDIT the following line; it is updated by the bump_version script (https://github.com/Azure/iot-sdks-internals/blob/master/release/csharp/inputs.js)
+            const string version = "1.5.1"; // PCLAssemblyVersion
```

Comment:
I'll update the bump_version script to search/replace in the updated file location, using an updated regex.